### ISSUE TITLE
Comments for `pooling_avg` is alias for include_padding or exclude_padding is confusing.

### ISSUE THREAD

**Issue by retonym** [oneapi-src/oneDNN#1069]
# Summary
Comments for `pooling_avg` is alias for `pooling_avg_include_padding` or `pooling_avg_exclude_padding` is confusing.
In dnnl.hpp, line 551 shows exclude, while line 552 shows include. 
Could you help to clarify it and make modifications for the comments. Thanks

# URLs
https://github.com/oneapi-src/oneDNN/blob/master/include/oneapi/dnnl/dnnl.hpp#L551-L552

**Comment by igorsafo** [comment]
Hi @retonym , thank you for your report. Indeed this is a typo. C API show the real value of `dnnl_pooling_avg`:
```cpp
/// Average pooling include padding
dnnl_pooling_avg_include_padding = 0x2ff,
/// Average pooling exclude padding
dnnl_pooling_avg_exclude_padding = 0x3ff,
/// Average pooling (alias for #dnnl_pooling_avg_exclude_padding)
dnnl_pooling_avg = dnnl_pooling_avg_exclude_padding,
```

Regards,
Igor Safonov

**Comment by igorsafo** [comment]
@retonym ,
The changes were promoted into our internal repository and will be available in public repository once we sync them (usually we sync them once per 2 weeks). I am reopening it till the changes are available in public repository.

Regards,
Igor Safonov

**Comment by retonym** [comment]
> @retonym ,
> The changes were promoted into our internal repository and will be available in public repository once we sync them (usually we sync them once per 2 weeks). I am reopening it till the changes are available in public repository.
> 
> Regards,
> Igor Safonov

Hi @igorsafo , thanks for your reply and code change.

**Comment by igorsafo** [comment]
Fixed in ebaa827

Regards,
Igor Safonov
